cmd/keytransparency-signer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang
+
+ENV DB_USER=test \
+    DB_PASSWORD=zaphod \
+    DB_DATABASE=test \
+    DB_HOST=127.0.0.0:3306
+
+ENV MAP_ID=0 \
+    MAP_URL=""
+ENV LOG_ID=0 \
+    LOG_URL=localhost:8090 \
+    LOG_KEY=genfiles/trillian-log-pubkey.pem
```

Comment:
OK, I'll change the default to `../trillian/testdata/log-rpc-server.pubkey.pem`.